NEWS

```diff
@@ -27,6 +27,13 @@ Improvements
 * A failing ``expectThat`` now fails tests run with
   ``AsynchronousDeferredRunTest``.  (Jonathan Lange, #1532452)
 
+* New ``testtools.twistedsupport`` module that collects all of our Twisted
+  support code in one place, including that currently available under
+  ``testtools.deferredruntest``.  (Jonathan Lange)
+
+* New matchers for testing ``Deferred`` code: ``failed``, ``successful``, and
+  ``NO_RESULT``.  (Jonathan Lange, Tristan Seligmann, #1369134)
```

Comment:
And the prevailing matcher naming convention suggests some other names, I think - `fails` and `succeeds`.  Also ... `continues` (or something) for `NO_RESULT` but that's probably 98% too cute.